Source/Utils.swift

```diff
@@ -314,6 +314,16 @@ class KeyboardHeightHelper: ObservableObject {
 
 #endif
 
+
+// MARK: - Hide keyboard
+
+func hideKeyboard() {
+#if os(iOS)
+    UIApplication.shared
```

Comment:
maybe better use @focusable - just to avoid UIKit references, what do you think?